bus/client.go

```diff
@@ -490,11 +490,16 @@ func (c *Client) SearchObjects(ctx context.Context, offset, limit int, key strin
 	return
 }
 
-// Object returns the object at the given path, or, if path ends in '/', the
-// entries under that path.
-func (c *Client) Object(ctx context.Context, path string) (o object.Object, entries []string, err error) {
+// Object returns the object at the given path with the given prefix, or, if
+// path ends in '/', the entries under that path.
+func (c *Client) Object(ctx context.Context, path, prefix string, offset, limit int) (o object.Object, entries []string, err error) {
```

Comment:
(nit) bit inconsistent with the other client methods where offset and limit comes after the context.

Reply:
I think it's weird to pass the `offset` and `limit` before passing the predicate but I changed it.